### DISCUSSION TITLE
Probably typos

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#1569] [Translations]
Dear @kajaji 

I think I found some typos in Al-Sufi skyculture:

I see "Al-Ayyuq", "Al-Ayyouq" and "Ayyuq" - probably this is same person and in this case it should use same name.

I see "Al-Thurayya" and "Al-Thurraya" - probably second name has typo, because it use once and I fixed the typo in the SC.

What do you think?

**Comment by axd1967** [comment]
and see also https://github.com/Stellarium/stellarium/pull/1419#issuecomment-749617989 concerning haab

**Comment by kajaji** [comment]
Thanks Alex,
yes "Al-Thurayya" is the correct one.
"Al-Ayyuq" and "Al-Ayyouq" are the same and we can use the first one "Al-Ayyuq".
"Ayyuq" should be left as it is because it is part of the complete name "Ayyuq Al-Thurayya". This is how it is in Arabic language.

May be for the next update, I should use transliteration for the proper names.

**Reply by alex-w** [reply]
Thanks!

OK, typo is fixed
